carto/sql.py

```diff
@@ -454,3 +455,14 @@ def copyto_file_path(self, query, path, append=False):
         file_mode = 'wb' if not append else 'ab'
         with open(path, file_mode) as f:
             self.copyto_file_object(query, f)
+
+    def copyto_stream(self, query):
+        """
+        Gets data from a table into a stream
+        :param query: The "COPY { table_name [(column_name[, ...])] | (query) }
+                           TO STDOUT [WITH(option[,...])]" query to execute
+        :type query: str
```

Comment:
We need to document the return type. Something like:

```
:return: the data output from COPY
:rtype: file-like object
```

Reply:
:+1: